assets/src/components/routeLadder.tsx

```diff
@@ -63,25 +67,49 @@ export const Header = ({
   return (
     <Card className="c-route-ladder__header">
       <Card.Body>
-        <div className="c-route-ladder__header__action-container">
+        <div
+          className={joinClasses([
+            "c-route-ladder__dropdown",
+            hasAlert && "non-skate-alert",
+          ])}
+        >
           {showDropdown && (
             <Dropdown className="border-box inherit-box">
               <Dropdown.Toggle
-                className="c-route-ladder__dropdown-button d-none d-sm-block"
+                className="c-route-ladder__dropdown-button d-none d-sm-flex"
                 aria-labelledby={joinTruthy([
                   routePillId,
                   routeOptionsToggleId,
                 ])}
               >
+                <ThreeDotsVertical />
                 <span className="visually-hidden" id={routeOptionsToggleId}>
                   Route Options
                 </span>
               </Dropdown.Toggle>
               <Dropdown.Menu>
+                <Dropdown.Header>
+                  <div className="c-route-ladder__dropdown-header-text">
+                    Adjustments
+                  </div>
+                </Dropdown.Header>
                 <Dropdown.Item className="icon-link" onClick={onClickAddDetour}>
-                  <PlusSquare />
-                  Add detour
+                  <PlusSquare /> Add detour
                 </Dropdown.Item>
+                {hasAlert && (
+                  <>
+                    <Dropdown.Divider />
```

Comment:
![Screenshot 2024-09-25 at 11 44 57 AM](https://github.com/user-attachments/assets/28224026-5c9e-4dca-bd30-3d521862c6c4)

The border here is a bit dark and heavy compared to the designs. I think the problem is both that it's the wrong color (`#808080` versus `gray-300`, which is `#d4d7db`) and that it's 2px instead of 1px.

You can fix the color by adding a `gray-300` mapping to `_map_overrides`, and then giving this element a `className` of `border-gray-300`. Then you can make it only 1px high by getting rid of the top or bottom border, like so:

```suggestion
                    <Dropdown.Divider className="border-gray-300 border-top-0" />
```

Result:

![Screenshot 2024-09-25 at 11 44 35 AM](https://github.com/user-attachments/assets/36576c40-1ced-4c09-84f8-5b1b244223f1)

(Caveat: adding a `gray-300` mapping to `_map_overrides` might be _evil_ in some way. At this point, I'm tempted to do it, because I'm frustrated by bootstrap's inflexibility, and want it to have more tailwind-style utilities, but I dunno, maybe that's evil. I think the alternative is more custom CSS.)

Reply:
Nixed my last comment, because I see the value of adding the gray-300 color change, though `_map_overrides` is (so far) just for the map. I'll find another place to make the utility

Reply:
I had to add another utility-class color to `_map_overrides` (`active-detour`, I think), so if/when you find a better place, let me know! 😅 

(Also, `_map_overrides` _works_, so if you want to avoid being blocked on finding the right home for this, then you could do `_map_overrides` for now as a stopgap, and move it later.)

Reply:
I didn't prefer adding to `_map_overrides` and didn't feel strongly about keeping this out of the `_route_ladders` scss, so [here](https://github.com/mbta/skate/pull/2802/commits/6b976ac5e0c03f6285d9ab5d614eb2cd85a30c50) was the change